src/main/resources/quests.json

```diff
@@ -1,2325 +1,2405 @@
+        "reqs": [
+            {
+                "skill": "Runecraft",
+                "level": "35",
+                "boostable": false
+            }
+        ]
+    },
+    {
+        "name": "Eagles' Peak",
+        "uri": "https://oldschool.runescape.wiki/w/Eagles%27_Peak",
+        "reqs": [
+            {
+                "skill": "Hunter",
+                "level": "27",
+                "boostable": false
+            }
+        ]
+    },
+    {
+        "name": "A Tail of Two Cats",
+        "uri": "https://oldschool.runescape.wiki/w/A_Tail_of_Two_Cats",
+        "reqs": []
+    },
+    {
+        "name": "Legends' Quest",
+        "uri": "https://oldschool.runescape.wiki/w/Legends%27_Quest",
+        "reqs": [
+            {
+                "skill": "Agility",
+                "level": "50",
+                "boostable": true
+            },
+            {
+                "skill": "Crafting",
+                "level": "50",
+                "boostable": false
+            },
+            {
+                "skill": "Herblore",
+                "level": "45",
+                "boostable": false
+            },
+            {
+                "skill": "Magic",
+                "level": "56",
+                "boostable": false
+            },
+            {
+                "skill": "Mining",
+                "level": "52",
+                "boostable": true
+            },
+            {
+                "skill": "Prayer",
+                "level": "42",
+                "boostable": false
+            },
+            {
+                "skill": "Quest points",
+                "level": "107",
+                "boostable": false
+            },
+            {
+                "skill": "Smithing",
+                "level": "50",
+                "boostable": false
+            },
+            {
+                "skill": "Strength",
+                "level": "50",
+                "boostable": false
+            },
+            {
+                "skill": "Thieving",
+                "level": "50",
+                "boostable": false
+            },
+            {
+                "skill": "Woodcutting",
+                "level": "50",
+                "boostable": false
+            }
+        ]
+    },
+    {
+        "name": "Land of the Goblins",
+        "uri": "https://oldschool.runescape.wiki/w/Land_of_the_Goblins",
+        "reqs": [
+            {
+                "skill": "Agility",
+                "level": "38",
+                "boostable": true
+            },
+            {
+                "skill": "Fishing",
+                "level": "40",
+                "boostable": false
+            },
+            {
+                "skill": "Thieving",
+                "level": "45",
+                "boostable": false
+            },
+            {
+                "skill": "Herblore",
+                "level": "48",
+                "boostable": false
+            },
+        ]

```

Comment:
Blows up locally due to missing `,` after the `]`. I re-wrote the parser and updated the README file in this repo in regards to this for future changes.